components/unit/filterable-table/UnitFilterableTable.vue

```diff
@@ -76,11 +76,21 @@
           <a target="_blank" rel="noreferrer noopener" :href="value"> Link </a>
         </template>
         <template
-          v-for="header in headers.filter(h => h.value !== 'url')"
+          v-for="header in headers.filter(
+            h => h.value !== 'url' && !header.hasOwnProperty('formatter')
+          )"
           #[`item.${header.value}`]="slotProps"
         >
           {{ formatItemAsString(slotProps) }}
         </template>
+        <template
+          v-for="header in headers.filter(header =>
+            header.hasOwnProperty('formatter')
+          )"
+          #[`item.${header.value}`]
```

Comment:
This line should be ```#[`item.${header.value}`]="{ value }"```

Otherwise everything is formatted as "Yes" :)

Reply:
Thanks ;) It's fixed now  !